src/templates/rust-all/LICENSE.tftpl

```diff
@@ -0,0 +1,100 @@
+Business Source License 1.1
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
+"Business Source License" is a trademark of MariaDB Corporation Ab.
+
+-----------------------------------------------------------------------------
+
+Parameters
+
+Licensor:             Arrow Air
```

Comment:
Are we Arrow Air or Arrow?

Reply:
Good question, we might even want to be Arrow DAO, let's check with Deetwo and Thomas

Reply:
As per discussion, let's use Arrow DAO for now.

Reply:
fixed: https://github.com/Arrow-air/tf-github/pull/71/commits/25dcc172a5ba717685ea88b906f19fb87a37f573